ext/gitdll/gitdll.c

```diff
@@ -254,14 +255,15 @@ int git_get_commit_next_parent(GIT_COMMIT_LIST *list, GIT_HASH hash)
 int git_free_commit(GIT_COMMIT *commit)
 {
 	struct commit *p = commit->m_pGitCommit;
+	char* pbuf = get_commit_buffer(commit, NULL);

```

Comment:
Should be ```char* pbuf = get_commit_buffer(p, NULL);```

Now, this PR is broken(can not be used).
Rework and climbed to git 2.0.1.